### ISSUE TITLE
Debugging with kill -s USR1 makes app hang

### ISSUE THREAD

**Issue by kevinob11** [balderdashy/sails#3369]
When I debug my app using `kill -s USR1 [pid]` the app hangs. I've tested with a brand new sails 0.11 app. I'm on node 12.7 on osx 10.11.1. This doesn't happen when I debug using `sails debug` or `node -debug app.js`. I know I can just use one of those, but there is a specific workflow that requires I start the debugger after I start sails.

**Comment by sailsbot** [comment]
Thanks for posting, @kevinob11.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
